ignite/contrib/handlers/lr_finder.py

```diff
@@ -136,6 +137,8 @@ def _reset(self, trainer: Engine) -> None:
     def _log_lr_and_loss(self, trainer: Engine, output_transform: Callable, smooth_f: float, diverge_th: float) -> None:
         output = trainer.state.output
         loss = output_transform(output)
```

Comment:
@vfdev-5 Just a question, do we really need this test ? I mean, if sequential, all_reduce should do nothing.